### PR TITLE
Stop making DB queries on console mode

### PR DESCRIPTION
**Pull request by MehediDracula**
AsgardCMS makes some unnecessary DB queries on console mode. Because of this behavior user has to run DB service on development environment even he only wants to run the artisan command

### CODE REVIEW

**Comment by nWidart** [comment]
Thanks!

Somehow your changes on the AuthController were sent here too though.

**Comment by MehediDracula** [comment]
yeah my bad :(

File: Modules/Core/Providers/AsgardServiceProvider.php [hunk 1]
```diff
@@ -12,7 +12,7 @@ class AsgardServiceProvider extends ServiceProvider
 {
     public function register()
     {
-        if (class_exists(TranslationServiceProvider::class)) {
+        if (! $this->app->runningInConsole() && class_exists(TranslationServiceProvider::class)) {
```
Comment (nWidart) [inline Modules/Core/Providers/AsgardServiceProvider.php]: Could you use `=== false` here to stay consistent with the codebase please?

File: Modules/Menu/Providers/MenuServiceProvider.php [hunk 2]
```diff
@@ -176,9 +176,13 @@ private function hasChildren($item)
      */
     private function registerMenus()
     {
-        if ($this->app['asgard.isInstalled'] === false || $this->app['asgard.onBackend'] === true) {
+        if ($this->app['asgard.isInstalled'] === false ||
+            $this->app['asgard.onBackend'] ||
+            $this->app->runningInConsole()
```
Comment (nWidart) [inline Modules/Menu/Providers/MenuServiceProvider.php]: Please use `=== true` for the 2 conditions above please. 😄

**Comment by nWidart** [comment]
I'm not sure how you can remove the AuthController changes. Maybe update your fork (upstream), but that might create a merge commit.

**Comment by MehediDracula** [comment]
May i should make another PR without this AuthController changes?

**Comment by nWidart** [comment]
The should work yes, but requires you to delete your current fork (and copy changes somewhere on your disk), re-fork, and re-apply the changes.

**Comment by MehediDracula** [comment]
ok... then maybe this PR should be closed

**Pull request closed without merging**